app/helpers/shared_helper/ab_list_helper.rb

```diff
@@ -0,0 +1,147 @@
+module SharedHelper::AbListHelper
+  private
+
+  def shared_button_summary(custom_button, sb_items, resolve, custom_button_record)
```

Comment:
can you change the variables here -
` shared_button_summary(record, sb_item, resolve, custom_button)`